packages/features/bookings/lib/handleConfirmation.ts

```diff
@@ -338,7 +338,67 @@ export async function handleConfirmation(args: {
         );
       })
     );
+
     await Promise.all(promises);
+
+    if (paid) {
+      let paymentExternalId: string | undefined;
+      const subscriberMeetingPaid = await getWebhooks({
+        userId: triggerForUser ? booking.userId : null,
+        eventTypeId: booking.eventTypeId,
+        triggerEvent: WebhookTriggerEvents.BOOKING_PAID,
+        teamId: booking.eventType?.teamId,
+      });
+      const bookingWithPayment = await prisma.booking.findFirst({
+        where: {
+          id: bookingId,
+        },
+        select: {
+          payment: {
+            select: {
+              id: true,
+              success: true,
+              externalId: true,
+            },
+          },
+        },
+      });
+      const successPayment = bookingWithPayment?.payment?.find((item) => item.success);
+      if (successPayment) {
+        paymentExternalId = successPayment.externalId;
+      }
+
+      const paymentMetadata = {
+        identifier: "cal.com",
+        bookingId,
+        eventTypeId: booking.eventType?.id,
+        bookerEmail: evt.attendees[0].email,
+        eventTitle: booking.eventType?.title,
+        externalId: paymentExternalId,
+      };
+      const bookingPaidSubscribers = subscriberMeetingPaid.map((sub) =>
+        sendPayload(sub.secret, WebhookTriggerEvents.BOOKING_PAID, new Date().toISOString(), sub, {
+          ...evt,
+          ...eventTypeInfo,
+          bookingId,
+          eventTypeId: booking.eventType?.id,
+          status: "ACCEPTED",
+          smsReminderNumber: booking.smsReminderNumber || undefined,
+          paymentId: bookingWithPayment?.payment?.[0].id,
+          metadata: {
+            ...(paid ? paymentMetadata : {}),
+          },
+        }).catch((e) => {
+          console.error(
+            `Error executing webhook for event: ${WebhookTriggerEvents.BOOKING_PAID}, URL: ${sub.subscriberUrl}`,
+            e
+          );
+        })
+      );
+
+      // I don't need to await for this
+      Promise.all(bookingPaidSubscribers);
```

Comment:
Don't await to complete this

Reply:
We don't need to do a Promise.all I guess, if we aren't waiting for the results.

Reply:
> We don't need to do a Promise.all I guess, if we aren't waiting for the results.

It's ok, we want them to start in parallel instead of cascading even if we don't wait for the response

Reply:
But calls would go in parallel already due to line 380. We don't need the result of any of those promises. So we don't need line number 400 at all.